catalog/views.py

```diff
@@ -10,10 +11,14 @@ def catalog_view_contacts(request):
     return render(request, 'catalog/contacts.html')
 
 
-# def contact(request):
-#     if request.method == 'POST':
-#         name = request.POST.get('name')
-#         phone = request.POST.get('phone')
-#         message = request.POST.get('message')
-#         return HttpResponse(f"Спасибо, {name}! Ваше сообщение получено.")
-#     return render(request, 'students/contact.html')
+def product_list(request):
+    products = Product.objects.all()
+    context = {'products': products}
+    return render(request, 'catalog/products.html', context)
+
+
+def product_detail(request, product_id=Product.id):
+    product = Product.objects.get(id=product_id)
```

Comment:
```suggestion
    product = get_object_or_404(Product, pk=product_id)
```
Иначе если продукт не найден упадет 500 ошибка - это плохо